template/renovate.json.j2

```diff
@@ -0,0 +1,35 @@
+{
+  "extends": [
+    "helpers:pinGitHubActionDigests"
+  ],
+  "labels": [
+    "dependencies"
+  ],
+  "prCreation": "not-pending",
+  "reviewers": [
+    "team:developers"
+  ],
+  "rollbackPrs": true,
+  "schedule": [
+    "after 5:00 and before 6:00 every weekday"
+  ],
+  "timezone": "Europe/Berlin",
+  "packageRules": [
+    {
+      "matchUpdateTypes": [
+        "patch"
+      ],
+      "groupName": "All dependencies (patch only)"
+    }
+  ],
+  "lockFileMaintenance": {
+    "enabled": true,
+    "schedule": [
+      "after 22:00 and before 6:00 every weekday"
+    ]
+  },
+  "github-actions": {
+    // Do not update GitHub Workflows that are owned by templating
+    "ignorePaths": {[ workflow_files.files | map(attribute='path') | map('replace', '.j2', '') | map('replace', base_dir+'/template/', '') | map('regex_escape') | to_json }]
```

Comment:
This renders to

` "ignorePaths": ["\\.github/workflows/daily_security\\.yml", "\\.github/workflows/reviewdog\\.yaml"]`

do we need to escape dots in there?

Reply:
Fixed